model_analyzer/result/result_manager.py

```diff
@@ -227,14 +229,13 @@ def _compile_measurements(self, measurements):
                     inference_table_key=self.model_inference_table_failing_key)
```

Comment:
You will need to add the model_name parameter [here](https://github.com/triton-inference-server/model_analyzer/blob/08deaa42f2067af9303fa99de6aa7f1f0fe236a4/model_analyzer/result/result_manager.py#L226) as well. None of the tests caught it unfortunately. The result, constraint, and metrics managers are difficult to unit test.

Reply:
I see. Good catch. I guess we should add some tests to `L0_config` that some experiments do not pass the constraints so that the other branch of the code is tested as well.

Reply:
I will add it in L0_constraints